recipes-kernel/linux/files/sos_5.4.cfg

```diff
@@ -1,3 +1,10 @@
 CONFIG_ACRN_GUEST=y
-
-# CONFIG_REFCOUNT_FULL is not set 
+# CONFIG_REFCOUNT_FULL is not set
+CONFIG_NET_VENDOR_STMICRO=y
+CONFIG_STMMAC_ETH=m
+CONFIG_STMMAC_PCI=m
+CONFIG_STMMAC_HWTS=y
+CONFIG_STMMAC_SELFTESTS=y
+CONFIG_DWXPCS=m
+CONFIG_MARVELL_PHY=m
+CONFIG_MARVELL_10G_PHY=m
```

Comment:
The last three above look unrelated. Also I can't find CONFIG_DWXPCS in mainline. Is it in ACRN kernel?

stmmac can be enabled by adding KERNEL_FEATURES_append = " features/net/stmicro/stmmac.scc"

Reply:
The device ACRN needs to support is using marvell phy and DWPCS is the PHY converter driver for DW xPCS IP.

Reply:
CONFIG_DWXPCS could be found in https://github.com/intel/linux-intel-lts/blob/5.4/yocto/